### ISSUE TITLE
Improvde WebSocket UI

### ISSUE THREAD

**Issue by Prinzhorn** [mitmproxy/mitmproxy#4662]
After v7 there are a couple of things that can be improved, I don't want them to be forgotten:

1. Whenever a new WebSocket message arrives the list jumps back to the top. That means you cannot read through all messages by scrolling down while the socket is open because it will keep jumping back (ttps://github.com/mitmproxy/mitmproxy/pull/4486#issuecomment-796578909)
2. Opcode.TEXT message should not be passed through `bytes_to_escaped_str` but render as the original string (new `text` property) (https://github.com/mitmproxy/mitmproxy/pull/4650#issuecomment-868676866)

**Comment by Kazeam** [comment]
Killing currently only works from the event hook, we should probably rig it up to work more generally.

**Comment by Prinzhorn** [comment]
@Kazeam yes, absolutely. I wasn't aware that you couldn't kill the flow at will, see #4711

**Comment by cupen** [comment]
5 years later...  Where is the WebSocket UI? Can I use it with `mitmweb`?
